src/components/Board.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { BoardInterface } from '../context/BoardsContext';
+import Cell, { CellInterface } from './Cell';
+
+interface BoardComponentInterface {
+    board: BoardInterface
+}
+const Board: React.FC<BoardComponentInterface> = function ({ board }) {
+  const gridMap = board.grid.map((row, idx) => (
+    row.map((cell : CellInterface, jdx : number) => (
+      <Cell
+        column={jdx}
+        row={idx}
+        shipName={cell.shipName}
+        isHit={cell.isHit}
+        isShip={cell.isShip}
+        key={`${Math.random()}`}
```

Comment:
Let's not use a random value for the key here. Check out the last bullet point under "Trade Offs": https://reactjs.org/docs/reconciliation.html#trade-offs

If we are sure that the grid will never be reordered and we don't have any stable unique values to work with (such as IDs), then I think it's fine to use indexes and keys. You can simply bypass the linter in this case by adding `// eslint-disable-next-line <name_of_rule>` above the line you want to disable.

Reply:
Thank you, fixed it!